.github/workflows/stubtest_stdlib.yml

```
@@ -44,6 +44,6 @@ jobs:
           cache: pip
           cache-dependency-path: requirements-tests.txt
       - name: Install dependencies
-        run: pip install $(grep mypy== requirements-tests.txt)
+        run: pip install typing_extensions $(grep ^pathspec requirements-tests.txt)
```

Same rationale as for the `daily.yml` workflow

```suggestion
        run: pip install -r requirements-tests.txt
```